hoot-core/src/main/cpp/hoot/core/io/HootApiDbReader.cpp

```diff
@@ -97,7 +97,7 @@ void HootApiDbReader::open(QString urlStr)
     if (_email == "")
     {
       throw HootException("If a map name is specified then the user email must also be specified "
-                          "via: " + emailKey());
+                          "via: " + ConfigOptions().getApiDbEmailKey());
```

Comment:
This is a little nit-pick but since the function `getApiDbEmailKey` is static there is less overhead/more efficient to call it directly instead of creating the `ConfigOptions` object, calling the function, and the destroying the object.
`                          "via: " + ConfigOptions::getApiDbEmailKey());`
^^^ more efficient/cleaner way to call the static function.  Also applies to the call in `HootApiDbWriter::_openDb()`